recipe/meta.yaml

```diff
@@ -20,12 +19,12 @@ requirements:
     - pip
     - pytest-runner
     - setuptools >=20.0
```

Comment:
I think setuptools should be one of the latest version https://github.com/PyCQA/astroid/blob/14a0d657809ab5b42e7d416b686953508e6d8b91/pyproject.toml#L2
Bu we do not  need so strict constraints. A plain `setuptools` will be ok